parent/pom.xml

```diff
@@ -1066,53 +1070,136 @@
 			</properties>
 		</profile>
 
+		<!-- JBIDE-23206
+			 To force a local build of a pull request to use current date/time when generating plugins and features,
+			 instead of using jgit timestamps (which could be in the past, and therefore result in stuff that Eclipse
+			 thinks is OLDER that a current N build, use this "R" profile:
+			 mvn verify -PR
+		-->
 		<profile>
-			<id>test-on-installed-app</id>
+			<id>R</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-packaging-plugin</artifactId>
+						<version>${tychoVersion}</version>
+						<configuration>
+							<timestampProvider>default</timestampProvider>
+						</configuration>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		
+		
+		<!-- IT Tests profiles -->
```

Comment:
IT = Integration Tests. Therefore IT Tests = Integration Tests Tests. 

Why not just call this "itests" or "integration-tests" ? Same question for the "it-tests" profile. Why not just "it" or "itests" ?